docs/src/manual/performance_tips.md

```diff
@@ -0,0 +1,47 @@
+# Performance tips
+
+There are 3 phases of building and solving a Convex.jl problem:
+
+1. Building the expression tree. This is everything that happens before `solve!` is called, e.g. creating variables, problems, constraints, and other expressions.
+2. Formulating the problem in MathOptInterface. This happens automatically when `solve!` is called, and (as long as `solve!` is called with `silent=false`) emits a log message when completed, containing the amount of time it took and the total amount of memory allocations.
+3. Solving the problem. This happens in `solve!` after the problem is formulated. When `silent=false`, typically solvers will print information about the problem and their progress when solving it.
+
+If you are experiencing performance issues, it is important to isolate which of these three steps is taking the most time. Typically, step (1) should be the fastest, then step (2), then step (3).
+
+* If building the expression tree (step 1 above) is the slowest part, there is likely a performance bug in Convex.jl. Please [file an issue](https://github.com/jump-dev/Convex.jl/issues/new) with a reproducible minimal example; it may be able to be quickly fixed.
+* If formulating the problem is much slower than solving it, there may be a performance issue in Convex.jl. See [Faster problem formulation: Avoid scalar indexing](@ref) below for one tip to speed things up.
+* If solving the problem is the slowest part, you may still be able to improve performance. See [Speeding up solve times: dualization](@ref) and [Speeding up solve times: conic-friendly formulations](@ref) below. Also consider trying other solvers. For large problems, first-order solvers like SCS and COSMO may be more performant (albeit with higher convergence tolerances) than second-order solvers like Clarabel and Hypatia.
+
+If you're struggling to improve performance, feel free to ask for help in the [community forum](https://jump.dev/forum). Before asking a question, make sure to read the post [make it easier to help you](https://discourse.julialang.org/t/psa-make-it-easier-to-help-you/14757), which contains a number of tips on how to ask a good question.
```

Comment:
this is duplicated from the main page, but I think it could be handy to have it called out here also